thorlcr/graph/thgraph.cpp

```diff
@@ -1812,7 +1812,7 @@ bool CGraphBase::wait(unsigned timeout)
 
 void CGraphBase::abort(IException *e)
 {
-    if (aborted)
+    if (aborted || !started)
```

Comment:
Is there any potential problem with a child query that is just about to start?

Reply:
you're right, if this is a container containing child graphs, then it should be allowed to recurse.
This change (and the similar on in CSlaveGraph::abort weres incidental to the actual fix (in CSlaveGraph::done), I'll remove these changes.